CONTRIBUTING.md

```diff
@@ -52,6 +52,8 @@ This is going to:
 
 You can visit the website at [http://localhost:9000](http://localhost:9000).
 
+TO modify the configuration listed in config-helper.ts (e.g server port) you need to modify the file `local-docker-config.json` in `voice-web/docker` and not create a `config.json` in the root of the project.
```

Comment:
small 'O' need at 'T0'. should we need to tell "not create a `config.json` " here ?

Reply:
oh sorry. you right

Reply:
done